client/lib/plans/constants.js

```diff
@@ -16,7 +16,14 @@ import {
 	PRODUCT_JETPACK_CRM_MONTHLY,
 } from 'calypso/lib/products-values/constants';
 
-export const JETPACK_REDIRECT_URL = 'https://jetpack.com/redirect/';
+// If JETPACK_CLOUD_REDIRECT_CHECKOUT_TO_WPADMIN is true, checkout will redirect to the site's wp-admin,
+// otherwise it will redirect to the JETPACK_REDIRECT_URL. Checkout references these constants in:
+// client/my-sites/checkout/composite-checkout/hooks/use-get-thank-you-url/get-thank-you-page-url.ts
+export const JETPACK_CLOUD_REDIRECT_CHECKOUT_TO_WPADMIN = true;
+export const JETPACK_REDIRECT_URL =
+	'https://jetpack.com/redirect/?source=jetpack-checkout-thankyou';
+export const redirectCloudCheckoutToWpAdmin = () =>
+	JETPACK_CLOUD_REDIRECT_CHECKOUT_TO_WPADMIN ? true : false;
```

Comment:
[Non-blocking] This code is fine as-is, I just wanted to drop some informative feedback :-)

One way to simplify this would be to get rid of the boolean expression like so, since the constant is a boolean value on its own:

```suggestion
export const redirectCloudCheckoutToWpAdmin = () =>JETPACK_CLOUD_REDIRECT_CHECKOUT_TO_WPADMIN;
```

An alternative -- more type-safe but less obvious -- option would be to say:

```suggestion
export const redirectCloudCheckoutToWpAdmin = () => !! JETPACK_CLOUD_REDIRECT_CHECKOUT_TO_WPADMIN;
```

Reply:
Yes! That is excellent, thanks Matt! 🙂